vars/utils.groovy

```diff
@@ -25,3 +25,12 @@ def getCosaDir(params = [:]) {
 def pathExists(path) {
     return shwrapRc("test -e ${path}") == 0
 }
+
+// Thin wrapper around SimpleTemplateEngine().
+//     tmpl_str    string  -- templated string
+//     bindings    map     -- variables to fill in
```

Comment:
I wonder if we should put some of the context from the commit message about why it was placed in this library in the comment here.

Reply:
No strong opinion. Even without the API restriction thing, I think it makes sense to have it here to share more easily.